src/components/molecules/modals/FollowModal/index.jsx

```diff
@@ -0,0 +1,238 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { Link } from "react-router-dom";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBModal,
+  MDBIcon,
+  MDBModalBody,
+  MDBBtn,
+  MDBModalHeader,
+  MDBRow,
+  MDBCol,
+} from "mdbreact";
+
+//> Stylesheet
+import "./follow.scss";
+//#endregion
+
+//#region > Components
+/** @class A modal that shows a list of follows */
+class FollowModal extends React.Component {
+  state = {
+    loggedUser: null,
+  };
+
+  componentDidMount = () => {
+    const { loggedUser } = this.props;
+
+    this.setState({
+      loggedUser: loggedUser,
+    });
+  };
+
+  componentDidUpdate = () => {
+    const { loggedUser } = this.props;
+
+    if (loggedUser !== this.state.loggedUser) {
+      this.setState({
+        loggedUser: loggedUser,
+      });
+    }
+  };
+
+  follow = (personToFollow) => {
+    if (!this.props.loggedUser?.anonymous) {
+      this.props.follow(personToFollow).then(() => {
+        let loggedUser = this.props.loggedUser;
+        let follows = [];
+
+        for (let count in loggedUser.person.follows) {
+          follows.push(loggedUser.person.follows[count]);
+        }
+
+        follows.push({ slug: "p-" + personToFollow });
+
+        loggedUser.person.follows = follows;
+
+        this.setState({ loggedUser });
```

Comment:
This should be reworked.

- Use the new state defined above.
- Then remove the for loop. This workaround is no longer required. (`loggedUser.follows` is readOnly)